docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -20,27 +20,28 @@ The B3 admin panel is accessible from the My Apps section of your store's dashbo
 
 With B3, you can customize the placement of B3 app elements, display text, styling, and checkout configurations. You also have access to the lifecycle methods for many B3 modules to inject your own JavaScript functions.
 
-Additionally, you can use the B3 REST API to create, read, update, and delete items such as orders, companies, addresses, payments, sales reps, and users. This API allows you to build your own services or integrate B3 with third-party business tools such as Salesforce.
+Additionally, you can use the B3 REST API to create, read, update, and delete items such as orders, companies, addresses, payments, sales reps, and users. This API allows you to build your own services or integrate B3 with third-party business tools, such as Salesforce.
+
+This section gives details on how to edit the DOM, CSS, and display text, and how to define JavaScript lifecycle methods. It assumes you are familiar with editing Stencil themes locally and pushing them to a sandbox or production store using the Stencil CLI.
```

Comment:
@traci707 the reason I went with `how to edit the DOM, CSS, and display text, and how to define JavaScript lifecycle methods` rather than `how to edit the DOM and CSS, display text, and define JavaScript lifecycle methods` is that the verbs `edit` and `define` should be in the same form. This is a list of two and an additional thing, disguised as a list of three.

I agree this is clunky. Other suggestions? I'll puzzle it over later & would love your input in the mix.

Reply:
@slsriehl We can leave it as it is. It's not a big deal.